portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,6 +15,8 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.*;
```

Comment:
Do you just need ArrayList from this package? If so I'd recommend not wildcarding the import. Having specific imports is more readable in that it is easier to understand exactly what is being pulled into the code. It also slightly increases compile time.